dGame/dComponents/SkillComponent.cpp

```diff
@@ -499,6 +499,25 @@ void SkillComponent::HandleUnmanaged(const uint32_t behaviorId, const LWOOBJID t
 	delete context;
 }
 
+void SkillComponent::HandleUnmanaged(const uint32_t behaviorId, const LWOOBJID target, LWOOBJID source)
+{
+	auto* context = new BehaviorContext(target);
```

Comment:
The constructor already sets the originator, just use `new BehaviorContext(originator)` and don't set the originator afterwards.